react/components/NumericStepper/README.md

```diff
@@ -127,6 +129,25 @@ initialState = {
       onChange={event => setState({ value4: event.value })}
     />
   </div>
+  <div className="mb5">
+    <NumericStepper
+      label="Disabled"
+      minValue={1}
+      disabled
+      value={state.value5}
+      onChange={event => setState({ value5: event.value })}
```

Comment:
Nitpick: remove this function (or change to m`()=>{}` since the element is disabled, therefore not accessible. It will ease the reading for future developers.